### PR TITLE
Assessment Page

### PR DESCRIPTION
**Pull request by plinionr3**
The client assessment page where they can evaluate their knowledges by their role and get the result level and the respective link.

### CODE REVIEW

File: src/stores/quiz/index.ts [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 import { quizResult, quizAnswers } from './state'
-import { initializeQuizAnswers, getQuizResult, setSelectedRole, getScoreResponse } from './actions'
+import { initializeQuizAnswers, getQuizResult, setSelectedRole } from './actions'
 import type { QuizAnswers } from '@/domain/QuizAnswer'
 import type { QuizResult } from '@/domain/QuizResult'
 
 export const useQuizStore = defineStore('quizAnswer', () => {
   const updateQuizResult = async (quizAnswer: QuizAnswers): Promise<void> => {
-    const scoreResponse: QuizResult = await getScoreResponse(quizAnswer)
+    const scoreResponse: QuizResult = await getQuizResult(quizAnswer)
```
Comment (vitorlofonseca) [inline src/stores/quiz/index.ts:9]: Why a method called "getQuizResult" is in actions and not in getters?

File: src/views/HomeView.vue [hunk 2]
```diff
@@ -1,32 +1,172 @@
...
+          v-for="item in rolesStore.roles"
+          :key="item.name"
+          :label="item.label"
+          :value="item.name"
+        />
+      </ElSelect>
+    </div>
+  </div>
+
+  <div v-if="role && role.knowledges && role.knowledges.length" class="Container">
```
Comment (vitorlofonseca) [inline src/views/HomeView.vue:17]:
```suggestion
  <div v-if="role && role.knowledges?.length" class="Container">
```

File: src/views/HomeView.vue [hunk 3]
```diff
@@ -1,32 +1,172 @@
...
-const fetchMessage = async (): Promise<string> => {
+onMounted(async () => {
+  await rolesStore.fetchRoles()
+  if (rolesStore.roles.length > 0) {
+    localSelectedRole.value = ''
+    handleRoleChange(rolesStore.roles[0])
+  }
+})
+
+watch(localSelectedRole, (newRoleName) => {
```
Comment (vitorlofonseca) [inline src/views/HomeView.vue:75]: You don't need this watch. There is an event called @change for select. Check in  element-plus documentation. Then you don't need the localSelectedRole neither

**Review by vitorlofonseca — APPROVED** [review]

**Review by PedroMagalhaes20 — APPROVED** [review]
Well done galera

**Pull request merged**